oggm/cfg.py

```diff
@@ -274,6 +274,7 @@ def initialize(file=None):
     PARAMS['invert_with_sliding'] = cp.as_bool('invert_with_sliding')
     _k = 'optimize_inversion_params'
     PARAMS[_k] = cp.as_bool(_k)
+    PARAMS['use_shape_factor'] = cp.as_bool('use_shape_factor')
```

Comment:
I think we should rename it to ``use_shape_factor_for_inversion``. It is likely that we will want to separate between inversion and dynamical model's factors.

For testing between the factors it might be useful to have this option as a str: ``'Huss'`` (or the original paper name), ``"Adhikari"`` or empty for no shape factor

Reply:
Yes, I already also thought about an option like this, but just did not spend more time on it. Will do so.